tests/e2e/specs/LicenseCode.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+
+    'Check if the main p tag with the class license-text is present': function(browser) {
```

Comment:
Please add tests to assert that the HTML and print attributions are also present.

Reply:
Yup, Done